plugin/vim-gh-line.vim

```diff
@@ -285,8 +319,11 @@ func! s:CgitUrl(origin)
                 \ 'g:gh_cgit_url_pattern_sub:' . string(g:gh_cgit_url_pattern_sub)
 endfunc
 
-noremap <silent> <Plug>(gh-line) :call <SID>gh_line('blob')<CR>
-noremap <silent> <Plug>(gh-line-blame) :call <SID>gh_line('blame')<CR>
+noremap <silent> <Plug>(gh-line) :call <SID>gh_line('blob', 0)<CR>
+noremap <silent> <Plug>(gh-line-blame) :call <SID>gh_line('blame', 0)<CR>
+
+command! GHIteractive call <SID>gh_line('blob', 1)
```

Comment:
This is great. 
Could we have commands for the non-interactive invocations too ? That way the link generation can be via a command too. 
For example 

```
noremap <silent> <Plug>(gh-line) :GH<CR>
command! GH call <SID>gh_line('blob', 0)
```

Reply:
sure, sounds good 👍